### ISSUE TITLE
LVC Notice email subscription: Can I filter out Mock Data LVC Notices?

### ISSUE THREAD

**Issue by keita-kawabe** [nasa-gcn/gcn.nasa.gov#823]
When I use https://gcn.nasa.gov web interface to manage my email subscription for GCN Notices, there seems to be no option to filter out test notices generated by LVC Mock Data Challenge (MDC) events. These mock events have event ID starting with MS, e.g. MS230418v, as opposed to S.

There is a "LVC_TEST" tick box, but that Notice type was discontinued by LIGO/Virgo. Checking/unchecking "LVC_TEST" doesn't seem to affect MDC notices.  

Is there a convenient way to receive all LVC notices except for MDC via email?

**Comment by lpsinger** [comment]
No, there's no way to filter out mock LIGO/Virgo/KAGRA notices from email notifications right now, since they are sent to the same notice type / topic as real events.

**Comment by keita-kawabe** [comment]
Thank you @lpsinger , understood.
